subprojects/docs/src/docs/userguide/introductionDependencyManagement.adoc

```diff
@@ -50,7 +62,7 @@ Gradle takes your dependency declarations and repository definitions and attempt
 
 * Given a required dependency, Gradle first attempts to resolve the _module_ for that dependency. Each repository is inspected in order, searching first for a _module descriptor_ file (POM or Ivy file) that indicates the presence of that module. If no module descriptor is found, Gradle will search for the presence of the primary _module artifact_ file indicating that the module exists in the repository.
 
-** If the dependency is declared as a dynamic version (like `1.+`), Gradle will resolve this to the newest available static version (like `1.2`) in the repository. For Maven repositories, this is done using the `maven-metadata.xml` file, while for Ivy repositories this is done by directory listing.
+** If the dependency is declared as a dynamic version (like `1.+`), Gradle will resolve this to the highest available static version (like `1.2`) in the repository. For Maven repositories, this is done using the `maven-metadata.xml` file, while for Ivy repositories this is done by directory listing.
```

Comment:
Is "static version" a term we use throughout the docs? (I am not aware of it)

If it isn't I would change "available static version" to "available version".

Reply:
I actually didn't change the content of "How dependency resolution works" so it must have been there before. In other places we use "concrete version". I will change it to that for now.